.travis.yml

```diff
@@ -6,5 +6,6 @@ directories:
 env:
 language: java
```

Comment:
we don't use Travis,

Reply:
The original repo does so this was updated to pass the CI for that

Reply:
we don't use Travis. You need to change this.

Reply:
Done